### DISCUSSION TITLE
Start Coding with our Intro to Javascript Launchpad Experience with Scrimba

### DISCUSSION THREAD

**Discussion by morganersery1** [community/community#58265] [GitHub Education]
### Hi everyone,

![Launchpad Javascript Twitter Card](https://github.com/community/community/assets/107881423/a0a49a40-ea92-4854-b46f-6918c469353e)

**What is a Launchpad?**
As a part of our suite of Experiences, Launchpads are small bites of content designed to get you from start to finish in under 20 minutes. Build confidence in your coding skills with this Launchpad which introduces you to Javascript by using [Scrimba](https://scrimba.com/)’s interactive interface that lets you to begin writing your own code right away.

**Dive in!**
If you’re new to CS, consider this new Experience your entry point to coding for the first time. Apply for your student benefits via Global Campus to gain access to this exclusive tutorial built by Student Developer Pack partner Scrimba. Then redeem their benefit to keep exploring.

**Comment by larrnahmad** [comment]
[](url)[](url)

**Comment by larrnahmad** [comment]
[adcookies.csv](https://github.com/community/community/files/11875179/adcookies.csv)

**Comment by Vana12330** [comment]
> ### Hi everyone,
> ![Launchpad Javascript Twitter Card](https://user-images.githubusercontent.com/107881423/246483133-a0a49a40-ea92-4854-b46f-6918c469353e.png)
> 
> **What is a Launchpad?** As a part of our suite of Experiences, Launchpads are small bites of content designed to get you from start to finish in under 20 minutes. Build confidence in your coding skills with this Launchpad which introduces you to Javascript by using [Scrimba](https://scrimba.com/)’s interactive interface that lets you to begin writing your own code right away.
> 
> **Dive in!** If you’re new to CS, consider this new Experience your entry point to coding for the first time. Apply for your student benefits via Global Campus to gain access to this exclusive tutorial built by Student Developer Pack partner Scrimba. Then redeem their benefit to keep exploring.

**Comment by jamessv** [comment]
great platform
